pkg/builder/jib.go

```diff
@@ -114,6 +114,8 @@ func (t *jibTask) Do(ctx context.Context) v1.BuildStatus {
 	if t.task.Configuration.ImagePlatforms != nil {
 		platforms := strings.Join(t.task.Configuration.ImagePlatforms, ",")
 		mavenArgs = append(mavenArgs, jib.JibMavenFromPlatforms+platforms)
+	} else {
+		mavenArgs = append(mavenArgs, jib.JibMavenFromPlatforms+"linux/amd64,linux/arm64")
```

Comment:
This is most likely going to break the upgrade check, as it happened in https://github.com/apache/camel-k/pull/5246#issuecomment-2009270960